README.md

```diff
@@ -55,15 +90,29 @@ JavaScript unit testing is done with [Jest](https://jestjs.io/). Along with Jest
 * [React Testing Library](https://testing-library.com/docs/react-testing-library/intro) for testing React.
 
 The tests are also automatically run when using the `npm run dev-server` command to start a development server.
+
 ### JavaScript Testing
 JavaScript unit testing is done with [Jest](https://jestjs.io/). Along with Jest, additional functionality is added with:
 * [jest-dom](https://github.com/testing-library/jest-dom) for testing the DOM.
 * [React Testing Library](https://testing-library.com/docs/react-testing-library/intro) for testing React.
+
 ### Java Testing
 Java unit testing is done with [JUnit 4](https://junit.org/junit4/).
+
 ### Adding tests
 **To add another JS test**, simply add a JS file that ends with `.test.js` (ex: `script.test.js`) and it will automatically be run by Jest.
 **To add another Java test**, add a Java file to the testing directory `src/test/java/`. Be sure to follow the Java conventions for package directory structure, for example if you're testing a package in `com.google.opensesame.github`, the test should be placed in the directory `src/test/java/com/google/opensesame/github`.
 
 ## File Structure
 Please refer to [this](https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html) resource to learn more about what each project directory should be used for, and take a look at [this section](https://maven.apache.org/guides/getting-started/#how-do-i-make-my-first-maven-project) of the Maven getting started guide to see how the project naming scheme affects directory structure.
+
+### Data Structures and Naming
+Opensesame only intends to store Data that can’t be sourced from GitHub in Datastore. When Data is sent to the front-end, we plan to conditionally populate Information sourced from GitHub. This is to conserve API calls.
```

Comment:
nit: The way this is worded makes it a bit confusing imo. Consider: 
```suggestion
Opensesame only intends to store Data in Datastore that can’t be sourced from GitHub. When Data is sent to the front-end, we plan to conditionally populate Information sourced from GitHub. This is to conserve API calls.
```

Reply:
Also, I don't think this is primarily to conserve API calls, and in fact it increases the number of API calls. It might be more accurate to mention that this avoids issues with data getting out of sync between GitHub and our local version.

I think this might be a wording issue, and I think you're referring to the fact that it is *conditionally* populated for why API calls are conserved. If that is the case, consider rewording or separating that into another paragraph.

Reply:
```suggestion
OpenSesame only intends to store data that can’t be sourced from GitHub in Datastore. When data is sent to the front-end, we plan to conditionally populate Information sourced from GitHub. This is to conserve API calls.
```